src/auth/screens/language-settings.js

```diff
@@ -1,23 +1,48 @@
 export default [
```

Comment:
Instead of sorting all languages by hard coding, how about invoking `Array.prototype.sort` at `language-setting.screen.js`? Then we can insert new languages without keeping sorting in mind.

Reply:
Good idea! Added with a new commit.